plugins/GigPlayer/CMakeLists.txt

```diff
@@ -3,14 +3,16 @@ if(LMMS_HAVE_GIG)
 	INCLUDE_DIRECTORIES(${GIG_INCLUDE_DIRS})
 
 	# Required for not crashing loading files with libgig
-	SET(GCC_COVERAGE_COMPILE_FLAGS "-fexceptions")
-	add_definitions(${GCC_COVERAGE_COMPILE_FLAGS})
+	if(NOT MSVC)
+		SET(GCC_COVERAGE_COMPILE_FLAGS "-fexceptions")
+		add_definitions(${GCC_COVERAGE_COMPILE_FLAGS})
+	endif(NOT MSVC)
```

Comment:
This is a leftover from #4000. All occurences of `-fexceptions` can now be removed.